packages/commonwealth/client/scripts/state/api/threads/getThreadById.ts

```diff
@@ -0,0 +1,17 @@
+import Thread from 'client/scripts/models/Thread';
+import { trpc } from 'client/scripts/utils/trpcClient';
+
+const THREAD_STALE_TIME = 5000; // 5 seconds
+
+const useGetThreadByIdQuery = (thread_id: number, enabled = true) => {
```

Comment:
do we need separate hook for that? can't we just use `getThreadsById` with single value